INSTALLATION.md

```diff
@@ -112,6 +116,36 @@ Octobox will be running on [http://localhost:3000](http://localhost:3000).
 
 **Note**: You can add `GITHUB_CLIENT_ID` and `GITHUB_CLIENT_SECRET` to a `.env` file instead of supplying them directly on the command-line.
 
+
+### Production environment
+
+First, Create a network interface 
+
+```bash
+docker network create octobox-network 
+```
+
+First, download and run postgres instance
```

Comment:
Two `First` steps

Reply:
thanks @knolleary I just fixed the typo and pushed it